proposals/0035-linalg-matrix.md

```diff
@@ -90,25 +93,37 @@ class Matrix {
   static typename hlsl::enable_if<hlsl::is_arithmetic<T>::value, Matrix>::type
   Load(/*groupshared*/ T Arr[], uint StartIdx, uint Stride, bool ColMajor);
 
+  template <MatrixUse UseLocal = Use>
+  typename hlsl::enable_if<Use == MatrixUse::A && Scope == MatrixScope::Wave &&
+                               UseLocal == Use,
+                           Matrix>::type
+      FromThreadVectors(vector<ElementType, MScalars>);
+
+  template <MatrixUse UseLocal = Use>
+  typename hlsl::enable_if<Use == MatrixUse::B && Scope == MatrixScope::Wave &&
+                               UseLocal == Use,
+                           Matrix>::type
+      FromThreadVectors(vector<ElementType, NScalars>);

```

Comment:
Did you purposely leave off `MatrixUse::Accumulator` here (and `GetThreadVector()` below)? If so, a comment as to why would be useful